src/services/tx/tx-sender/dispatch.ts

```diff
@@ -191,37 +200,39 @@ export const dispatchBatchExecution = async (
     const provider = createWeb3(wallet.provider)
     result = await multiSendContract.contract.connect(provider.getSigner()).multiSend(multiSendTxData, overrides)
 
-    txs.forEach(({ txId }) => {
-      txDispatch(TxEvent.EXECUTING, { txId, groupKey })
+    txs.forEach(({ txId, txInfo: { humanDescription } }) => {
```

Comment:
Do we need a fallback for this dispatcher?

Reply:
I've changed it to a static description `Batch transaction`. Imo it is excessive to show a notification for every transaction but this is something we could also change later.